Romo/lib/Drive/RomoNetworking/Low Latency/RMDataSocket.m

```diff
@@ -10,7 +10,7 @@
 
 #pragma mark - Constants --
 
-#define SEND_BUFFER_SIZE 32000
+#define SEND_BUFFER_SIZE 65535
```

Comment:
Why the change in the buffer size?

Reply:
Increase the buffer size to UDP maximum limit in order to stream better qualify video.